src/main/java/com/client/core/base/util/Utility.java

```diff
@@ -23,34 +48,12 @@
 import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
-import javax.xml.datatype.DatatypeConfigurationException;
-import javax.xml.datatype.DatatypeFactory;
-import javax.xml.datatype.XMLGregorianCalendar;
-
-import com.client.core.base.model.relatedentity.BullhornRelatedEntity;
-import com.client.core.base.model.relatedentity.StandardRelatedEntity;
-import com.client.core.base.workflow.node.WorkflowAction;
-import com.google.common.collect.Maps;
-import com.google.common.collect.Sets;
-import org.apache.commons.lang3.StringUtils;
-import org.joda.time.DateTime;
-import org.joda.time.DateTimeZone;
-import org.joda.time.Days;
-import org.json.JSONArray;
+public class Utility {
 
-import com.bullhornsdk.data.api.BullhornData;
-import com.bullhornsdk.data.model.entity.core.type.BullhornEntity;
-import com.bullhornsdk.data.model.entity.core.type.QueryEntity;
-import com.bullhornsdk.data.model.entity.core.type.SearchEntity;
-import com.bullhornsdk.data.model.entity.embedded.Address;
-import com.bullhornsdk.data.model.parameter.QueryParams;
-import com.bullhornsdk.data.model.parameter.SearchParams;
-import com.bullhornsdk.data.model.parameter.standard.ParamFactory;
-import com.bullhornsdk.data.model.response.list.ListWrapper;
-import com.client.core.AppContext;
-import com.google.common.collect.Lists;
+    private static final int BATCH_SIZE = 200;
 
-public class Utility {
+    @Autowired
+    private static BullhornData BULLHORN_DATA;
```

Comment:
This might be strange to actually be Autowiring a dependency for a Utility class. I might have to look into another methodology for this